test/e2e/util.go

```diff
@@ -103,19 +104,28 @@ func waitSizeReachedWithAccept(t *testing.T, f *framework.Framework, clusterName
 		}
 		names = nil
 		var nodeNames []string
+		var podIPs []string
 		for i := range podList.Items {
 			pod := &podList.Items[i]
-			if pod.Status.Phase != v1.PodRunning || !acceptPod(pod) {
+			if pod.Status.Phase != v1.PodRunning || len(pod.Status.PodIP) == 0 || !acceptPod(pod) {
 				continue
 			}
 			names = append(names, pod.Name)
 			nodeNames = append(nodeNames, pod.Spec.NodeName)
+			podIPs = append(podIPs, pod.Status.PodIP)
 		}
 		logfWithTimestamp(t, "waiting size (%d), etcd pods: names (%v), nodes (%v)", size, names, nodeNames)
 		if len(names) != size {
 			return false, nil
 		}
-		// TODO: check etcd member membership
+		// When this method returns true, we need to make sure all members are up and running.
```

Comment:
this change makes the function bloated. originally, the function only cares about pod status. Now it also cares about the etcd member status. we should put the member checking logic into the acceptFunc.